client/src/styles/legacy/_tree.scss

```diff
@@ -312,6 +312,9 @@
           background-image: linear-gradient(to bottom, #3875d7 20%, #2a62bc 90%);
           color: #FFF;
           border: none;
+          .jstree-pageicon {
```

Comment:
I assume this is a specificity override as the colour is defined a couple of lines up. Do you want to add a comment about that?